python/kadalu_content_apis/objects.py

```diff
@@ -68,40 +68,40 @@ def upload(cls, conn, bucket_name, file_path, object_type, path, immutable, vers
 
 
     @classmethod
-    def list(cls, conn, bucket_name):
-        """ List object(s) of both default("/") and with bucket-name """
+    def list(cls, conn, folder_name):
+        """ List object(s) of both default("/") and with folder-name """
 
-        if bucket_name == "/":
+        if folder_name == "/":
```

Comment:
same here, else two slashes comes in Line 77

Reply:
Updated